index.html

```diff
@@ -0,0 +1,161 @@
+            <div class="">
+              <ul>
+                <li><a href="index.html">Gaming convention 2022</a></li>
+              </ul>
+            </div>
+            <div class="navs-two">
+              <ul>
+                <li><a href="about.html">About</a></li>
+                <li><a href="#program">Program</a></li>
+                <li><a href="">Join</a></li>
+                <li><a href="">Sponsor</a></li>
+                <li><a href="">News</a></li>
+              </ul>
+            </div>
+            <div class="campaign">
+              <ul>
+                <li><a href="#">CC Campaign</a></li>
+              </ul>
+            </div>
+          </nav>
+        </div>
+      </header>
+      <main>
+        <section id="header-content">
+          <h1>Welcome All!</h1>
+          <div class="bar-game">
+            <h2>Barbados Gaming Convention 2022</h2>
+          </div>
+          <div class="paragraph">
+            <p class="paragraph-one">
+              Come and enjoy some of the world's best gaming consoles and
+              computer gaming desktops built by our very own, enjoy the light
+              show and play the latest games. Compititions will also be held
+              with amazing prizes.
+            </p>
+          </div>
+          <div class="date">
+            For Two days, May-14th-2022 until (Sun)May-15th-2022
+          </div>
+          <p class="sentence-one">@ Full Scale Gaming Auditorium</p>
+        </section>
+        <section id="program">
+          <h3>Main Program</h3>
+          <div class="program-block">
+            <div class="gaming one">
+              <i class="fa-solid fa-chess-board"></i>
+              <h3>Retro Games</h3>
+              <p>play old school games, relive your childhood</p>
+            </div>
+            <div class="gaming">
+              <i class="fa-solid fa-chess-board"></i>
+              <h3>Gaming</h3>
+              <p>play old school games, relive your childhood</p>
+            </div>
+            <div class="gaming">
+              <i class="fa-solid fa-chess-board"></i>
+              <h3>Cosplay</h3>
+              <p>play old school games, relive your childhood</p>
+            </div>
+            <div class="gaming">
+              <i class="fa-solid fa-chess-board"></i>
+              <h3>Gaming</h3>
+              <p>play old school games, relive your childhood</p>
+            </div>
+            <div class="gaming">
+              <i class="fa-solid fa-chess-board"></i>
+              <h3>Gaming</h3>
+              <p>play old school games, relive your childhood</p>
+            </div>
+          </div>
+          <div class="see-program">
+            <a href="#">SEE THE WHOLE PROGRAM</a>
+          </div>
+        </section>
+        <article id="feature">
+          <h3>Featured Speakers</h3>
+          <span class="underline"></span>
+          <div id="feature-block"></div>
+        </article>
+        <section class="partners">
```

Comment:
- [ ] The partner section is missing icons, please add them.

- [ ] Also make the width 100% on the parent container so as to get rid of the white space on the left and right of the main container, holding the partner's section.

Your design
![image](https://user-images.githubusercontent.com/81932478/169436012-cc483ff8-b723-40b2-a53a-bffdbe2a51fe.png)

Provided design
![image](https://user-images.githubusercontent.com/81932478/169436067-54db1685-5471-4add-838c-1b37561f68f2.png)

Reply:
partner icons were always there

Reply:
Implemented 👍